src/device/router_device_routing.erl

```diff
@@ -666,8 +666,7 @@ maybe_multi_buy(Offer, Attempts, Device) ->
             end
     end.
 
--spec lookup_mb(binary()) ->
-    {ok, binary(), non_neg_integer(), integer()} | {error, not_found}.
+-spec lookup_mb(binary()) -> {ok, binary(), non_neg_integer(), integer()} | {error, not_found}.
```

Comment:
FYI: when there are multiple items within a return value like this, Dialyzer accommodates adding variable names to the result type-- just like with function arguments.

Let's try doing this for new & modified code while it's fresh in our minds.